test/cjk.test.ts

```diff
@@ -0,0 +1,27 @@
+import {exec} from 'child_process';
+import * as fs from 'fs';
+import * as os from 'os';
+import DoneCallback = jest.DoneCallback;
+
+describe('Parse codes that contains CJK characters', () => {
+
+    it('generate PlantUML for classes that contains CJK characters', (done: DoneCallback) => {
+        // tslint:disable-next-line:max-line-length
+        exec('ts-node --project ./tsconfig.json ./src/index.ts -i ./test/CJK/CJK.ts --output ./output.puml', () => {
```

Comment:
This test is failing intermittently when it takes longer than 5 seconds to run.

Reply:
Never met it from my local machine, would increase the jest timeout to 10s fixed it?